.gitignore

```diff
@@ -17,4 +17,7 @@
 # Maven
 log/
 target/
-dependency-reduced-pom.xml
\ No newline at end of file
+dependency-reduced-pom.xml
+
+# Own schematics loader (not used currently but can be useful later; ugly as hell)
+plugin/src/main/java/com/rojel/wesv/SchematicsHandler.txt
```

Comment:
This schematics loader was completely written from scratch by me, as I thought the standard schematics loader was the issue that loaded schematics weren't displayed with particles. After 3h I saw my mistake. So this is just left over. Nevertheless I decided to not commit this ugly file.